### PR TITLE
Tweaks for SR and AZ

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR moves one facility from AZ's Test site to their Prod site, per their request.   In addition, it adds a value to Specimen_type Valueset, to support a change in SimpleReport.

This one additional value resulted in a zillion schema docs getting updated.

## Checklist
- [X] Tested locally?
- [ ] Added new dependencies?
- [X] Updated the docs?
- [X] Added a test?
- [X] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### CODE REVIEW

File: prime-router/docs/schema_documentation/az-az-covid-19-hl7.md [hunk 1]
```diff
@@ -2079,6 +2079,7 @@ Code | Display
 
 Code | Display
 ---- | -------
+445297001|Swab of internal nose
```
Comment (jimduff-usds) [inline prime-router/docs/schema_documentation/az-az-covid-19-hl7.md:2082]: This is the cause of all the ruckus.

File: prime-router/metadata/organizations-local.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@
         schema: az/az-covid-19
         jurisdictionalFilter:
           - matches(ordering_facility_state,AZ)
-          - "doesNotMatch(ordering_facility_name,Tucson Mountains)"
+          - "doesNotMatch(ordering_facility_name,Tucson Mountains,Tucson Foothills)"
         transforms: { deidentify: false }
```
Comment (jimduff-usds) [inline prime-router/metadata/organizations-local.yml:30]: And this is the other change - the foothills are joining the mountains.

File: prime-router/quick-test.sh [hunk 3]
```diff
@@ -26,6 +26,12 @@ RUN_FL=0
 RUN_STANDARD=1
 RUN_ALL=0
 
+# If no args, run everything.
+if [ $# -eq 0 ] 
+then
+  RUN_ALL=1
+fi
+
```
Comment (jimduff-usds) [inline prime-router/quick-test.sh:34]: This now runs "ALL" by default.

File: prime-router/src/test/csv_test_files/expected/simplereport-az.csv [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+"CDC PRIME - Atlanta, Georgia (Dekalb)",2.16.840.1.114222.4.1.237821,"Org1, Lab1",42D4513038,4149 Kelly Mills,Bertfurt,TX,23707-6278,8063832361,1,Geralyn,Grimes,,,,,,,AZ,,,,"Org1, Flat Plains",3562 Lockman Tunnel,Darrylberg,AZ,59206,,,,987379,20201208,Swab of internal nose,Blood specimen (specimen),94563-4,SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay,Not detected,20201214,
 "CDC PRIME - Atlanta, Georgia (Dekalb)",2.16.840.1.114222.4.1.237821,"Org1, Lab2",20D5239501,89794 Pamelia Burg,East Jeaniceton,FL,13431,1750558668,2,Junior,Stroman,19731223,O,O,U,8721 Corwin Vista,Lake Sharri,FL,46406,Reeves,5952132283,"Org1, Happy Valley",3672 Elayne Well,Ulyssesfort,AZ,61713-3494,,,,220690,20201209,Whole blood sample,Blood specimen (specimen),94563-4,SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay,Detected,20201206,
```
Comment (jimduff-usds) [inline prime-router/src/test/csv_test_files/expected/simplereport-az.csv:3]: Nasal Passages:   Tweaked the test dataset to include the new value.

**Review by cglodosky — APPROVED** [review]

**Pull request merged**